README.md

```diff
@@ -1 +1 @@
-# Project-Review
```

Comment:
Add a detailed readme file using the readme template that was shared

Reply:
Good point @ifnotlily, but let's always state the reasons WHY we want students to make a certain change because it gives them more insights on why your requested change is a better approach. 

And I think you would sound a bit more polite saying **_"I think if you add a detailed readme file to your project will give anyone visiting the repo first-hand information about what the project is about as well as all installation guide author(s) and ...."_**
With this, you wouldn't be sounding like you're issuing a command to the student.